resources/views/community/components/event/aotw.blade.php

```diff
@@ -1,31 +1,49 @@
+@props([
+    'achievement' => [],
+    'game' => [],
+    'consoleName' => '',
+    'forumTopicId' => 0,
+])
+
 <?php
+$achievementId = $achievement->ID;
+$achievementName = $achievement->Title;
+$achievementDescription = $achievement->Description;
+$achievementPoints = $achievement->Points;
+$achievementRetroPoints = $achievement->TrueRatio;
+$achievementBadgeName = $achievement->BadgeName;
 
-use App\Site\Models\StaticData;
+$renderedGameTitle = renderGameTitle($game->Title);
+$achievementIconSrc = media_asset("/Badge/$achievementBadgeName.png");
+$gameSystemIconUrl = getSystemIconUrl($game->ConsoleID);
+?>
 
-$staticData = StaticData::first();
+<div class="component">
+    <h3>Achievement of the Week</h3>
 
-if ($staticData === null) {
-    return;
-}
+    <div class="bg-embed p-4 rounded border border-embed-highlight">
+        <a href="{{ route('achievement.show', $achievement->ID) }}" class="group flex gap-x-2 mb-3 text-text transition-transform hover:-translate-y-0.5">
+            <img src="{{ $achievementIconSrc }}" alt="Achievement of the week badge" width="64" height="64" class="w-16 h-16">
+            <div>
+                <p class="font-semibold leading-4 text-link group-hover:text-link-hover">{{ $achievement->Title }}</p>
+                <p class="text-xs mb-1">5 <span class="TrueRatio">(12)</span> Points</p>
+                <p class="text-xs">{{ $achievement->Description }}</p>
+            </div>
+        </a>
 
-$achID = $staticData['Event_AOTW_AchievementID'];
-$forumTopicID = $staticData['Event_AOTW_ForumID'];
+        <hr class="border-embed-highlight mt-2 mb-3">
 
-$achData = GetAchievementData($achID);
-if (empty($achData)) {
-    return;
-}
-?>
-<div class="component">
-    <h3>Achievement of the Week</h3>
-    <div class="text-center">
-        <div>
-            {!! achievementAvatar($achData) !!}
-        </div>
-        in
-        <div>
-            {!! gameAvatar($achData, iconSize: 24) !!}
-        </div>
-        <a class="btn mt-2" href="/viewtopic.php?t={{ $forumTopicID }}">Join this tournament!</a>
+        <a href="{{ route('game.show', $achievement->GameID) }}" class="group flex gap-x-2 text-text transition-transform hover:translate-x-0.5">
+            <img src="{{ media_asset($game->ImageIcon) }}" alt="Achievement of the week game badge" width="32" height="32" class="w-8 h-8">
+            <div class="-mt-1">
+                <p class="font-semibold mb-0.5 text-xs text-link group-hover:text-link-hover">{!! $renderedGameTitle !!}</p>
+                <div class="flex items-center gap-x-1">
+                    <img src="{{ $gameSystemIconUrl }}" width="18" height="18" alt="Console icon">
+                    <span class="block text-xs tracking-tighter">{{ $consoleName }}</span>
+                </div>
+            </div>
+        </a>
```

Comment:
Something like this may eventually be worth moving into `achievementAvatar()` as an optional rendering mode, almost in the vein of some of the work happening in https://github.com/RetroAchievements/RAWeb/pull/1500. There are a few places elsewhere on the site this design (console on 2nd line with icon) makes sense, though it doesn't make sense universally.